src/commands/load-all.ts

```diff
@@ -30,6 +30,14 @@ function getRegister<F extends Register.Flags | Register.Flags[]>(
   return (argument.register = register as any);
 }
 
+function getInputLength(argument: { inputLength?: number }) {
+  const len = +(argument.inputLength as any);
+  if (len > 0 && Number.isInteger(len)) {
+    return len;
+  }
+  throw new ArgumentError('"inputLength" must be positive integer', "inputLength");
+}
```

Comment:
I believe this function is still unused?

You probably wanted to automatically use this function for `inputLength` arguments, similarly to what's done with [`count` and `repetitions`](https://github.com/71/dance/blob/e15330afbc365e1138a4ba7b1a9ce9fd379e64cc/src/commands/load-all.build.ts#L146-L154)?